libraries/libraries_api_sdk.go

```diff
@@ -0,0 +1,76 @@
+package libraries
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/apierr"
+	"github.com/databricks/databricks-sdk-go/service/compute"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+// clusterID string, timeout time.Duration, isActive bool, refresh bool
+func WaitForLibrariesInstalledSdk(ctx context.Context, w *databricks.WorkspaceClient, wait compute.Wait, timeout time.Duration) (result *compute.ClusterLibraryStatuses, err error) {
+	err = resource.RetryContext(ctx, timeout, func() *resource.RetryError {
+		libsClusterStatus, err := w.Libraries.ClusterStatusByClusterId(ctx, wait.ClusterID)
+		if err != nil {
+			var apiErr *apierr.APIError
+			if !errors.As(err, &apiErr) {
+				return resource.NonRetryableError(err)
+			}
+			if apiErr.StatusCode != 404 && strings.Contains(apiErr.Message,
+				fmt.Sprintf("Cluster %s does not exist", wait.ClusterID)) {
+				apiErr.StatusCode = 404
+			}
+			return resource.NonRetryableError(apiErr)
+		}
+		if !wait.IsRunning {
+			log.Printf("[INFO] Cluster %s is currently not running, so just returning list of %d libraries",
```

Comment:
I copied this from the existing funciton but this logging statement seems to be inaccurate? I think instead of saying "Cluster is currently not running", we should probably print something like "we don't have to wait for pending libraries, so just returning the list of libraries"